Sources/Confidence/RemoteConfidenceClient.swift

```diff
@@ -46,11 +46,14 @@ public class RemoteConfidenceClient: ConfidenceClient {
             switch result {
             case .success(let successData):
                 let status = successData.response.statusCode
+                let eventNames = events.map { event in
```

Comment:
These are the events that we intended to upload, not necessarily what the backend accepted, right?

Reply:
Good catch! Addressed